lib/data/odata.js

```diff
@@ -134,7 +134,9 @@ const submissionToOData = (fields, table, submission, options = {}) => new Promi
         attachmentsPresent: attachment.present || 0,
         attachmentsExpected: attachment.expected || 0,
         status: encrypted ? (encHasData ? 'notDecrypted' : 'missingEncryptedFormData') : null,
-        reviewState: submission.reviewState || null
+        reviewState: submission.reviewState || null,
+        deviceId: submission.deviceId || null,
+        edits: submission.aux.edit.count
```

Comment:
Can this be 0, or will it be `null`? I thought it would be `null` and made a related commit in getodk/central-frontend#430, but I think I might have misunderstood the comment about the Edits column of the CSV.